airflow/sensors/base_sensor_operator.py

```diff
@@ -70,6 +70,12 @@ def __init__(self,
                  *args,
                  **kwargs):
         super(BaseSensorOperator, self).__init__(*args, **kwargs)
+        if not isinstance(poke_interval, (int, float)) or poke_interval < 0:
```

Comment:
Do you think it's worth move it to a separate method?Operators for GCP have validation in a separate method.
Reference: https://github.com/apache/airflow/blob/master/airflow/contrib/operators/gcp_sql_operator.py#L166-L170

Reply:
If it's simple logic & is not going to be re-used for multiple times, I don't think it's necessary to have a separate method.

Reply:
Agree with @XD-DENG, but it may be worth moving to separate method, in a sense that there are three checks for the argument validation and moving them to separate method may simplify the `__init__ ` in terms of readability or say from a cleaner code perspective.

What you guys say? @XD-DENG @mik-laj

Reply:
In my opinion, if it is possible to say about a certain fragment of the code that it has a certain role, it is worth creating a method from this fragment. There are several reasons. First, the readability of the code. It is not complex, but the work on readability is a continuous process. In other places in the project, there are more complex methods that are better candidates for division, but we have to start with something. Second, small methods have the role of additional documentation. Third, this is just a practice that we follow in my team and it is worth spreading it. This is not a new idea that does not exist. This is the choice between two equivalent ways.

It is worth remembering that this is a small matter.  This should not prevent from merging, but only serve as a guide on how to write a code.  If you have time and you want it, you can improve it. This code is good enough to be accepted without changes.

Reply:
Sure, fair enough. @kani5hk please use the code chunk @mik-laj shared above as reference.

Thanks!